robot/rospackages/src/task_handler/scripts/start_ros_stream.sh

```diff
@@ -27,4 +27,22 @@ fi
 # run the cv_camera_node
 # web_video_server node must be running in background for this to work
 # the image width & image height are hard coded to meet the requirements of ERC 2K19
-rosrun cv_camera cv_camera_node _device_path:=$port _image_width:=1280 _image_height:=720
+# the name of then node can't contain "/" so only use the suffix, i.e. remove preceding "/dev/"
+# so in local mode you will get topic names like 'video0Cam', 'video1Cam', and so on
+# in competition mode you will get specifically 'FrontCam', 'RearCam', and 'ArmScienceCam'
+topicName="${port:5}Cam"
+
+# These results are from a test which was run with 3 logitech c920
+# all connected through the same usb hub to the 2.0 hub port of the odroid xu4
+# Last test done on: 26/12/2019
+# +---------------------------------+--------------------------+
+# |resolution --> (width x height)  | max simultaneous streams |
+# |---------------------------------|--------------------------|
+# |240p  --> 352x240                | 3                        |
+# |360p  --> 480x360                | 2                        |
+# |480p  --> 858x480, SD            | 1                        |
+# |720p  --> 1280x720, HD           | 1                        |
+# |1080p --> 1920x1080, full HD     | 0                        |
+# +---------------------------------+--------------------------+
```

Comment:
Should be moved to the wiki (?)

Reply:
I think that if we're at comp and we don't have internet it would be super useful to have this ready to read in the relevant place, in case something bad happens during comp. We should discuss more how to go about documenting our stuff in an organized manner. I think we should have most of it on the drive.